cmd/gotk/get_helmrelease.go

```diff
@@ -51,37 +54,43 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
```

Comment:
```suggestion
	table.SetHeader([]string{"Name", "Revision", "Ready", "Message", "Suspended"})
```

Given the user already specifies a mandatory namespace selector, as I do not think that displaying the namespace offers any additional value (`kubectl` as an example behaves the same, but _does_ display the name if the `--all-namespaces` flag is passed).

Reply:
Just to clarify, you're thinking to only remove the `Namespace` column in `get_helmrelease.go` correct? or remove the `Namespace` column in every `get` command?

Reply:
Sorry if my comment was vague, I meant:

> remove the `Namespace` column in every `get` command?

When the namespace flag is passed, it should however show for every `get`  command, hope this makes it more clear.

Reply:
Gotcha, thoughts on the new changes?